lighthouse-core/audits/legacy-javascript.js

```diff
@@ -217,41 +211,16 @@ class LegacyJavascript extends Audit {
       ['Reflect.isExtensible', 'es6.reflect.is-extensible'],
       ['Reflect.ownKeys', 'es6.reflect.own-keys'],
       ['Reflect.preventExtensions', 'es6.reflect.prevent-extensions'],
-      ['Reflect.set', 'es6.reflect.set'],
       ['Reflect.setPrototypeOf', 'es6.reflect.set-prototype-of'],
-      ['Set', 'es6.set'],
       ['String.prototype.codePointAt', 'es6.string.code-point-at'],
-      ['String.prototype.endsWith', 'es6.string.ends-with'],
       ['String.fromCodePoint', 'es6.string.from-code-point'],
-      ['String.prototype.includes', 'es6.string.includes'],
       ['String.raw', 'es6.string.raw'],
       ['String.prototype.repeat', 'es6.string.repeat'],
-      ['String.prototype.startsWith', 'es6.string.starts-with'],
-      ['String.prototype.trim', 'es6.string.trim'],
-      // These break the coreJs2/coreJs3 naming pattern so are set explicitly.
-      {name: 'ArrayBuffer', coreJs2Module: 'es6.typed.array-buffer', coreJs3Module: 'es.array-buffer.constructor'},
```

Comment:
aren't these all the biggest ones? and they're really all included in every esmodules: true bundle? :(

Reply:
:(